usagov/vote-gov-drupal
# VOTE-2524: Update favicon size

## Jira ticket

[VOTE-2524](https://cm-jira.usa.gov/browse/VOTE-2524)

## Description

Update the favicon to be the correct (full) size.

## Deployment and testing

### Post-deploy steps

1. lando retune

### QA/Testing instructions

1. Open vote-gov.lndo.site and check that the favicon appears in its full (larger) size. See below for comparison.
Before:
![image](https://github.com/user-attachments/assets/2780d47e-dcfd-45d9-96dc-d4457fc2080e)

After:
![image](https://github.com/user-attachments/assets/a1f7bc2b-f4cd-4b15-9425-c21e4bf41c75)

## Checklist for the Developer


## Checklist for the Peer Reviewers
